content/docs/06.enterprise/02.governance/read-only-secrets.md

```diff
@@ -0,0 +1,173 @@
+
+![read-only-secrets-1](/docs/enterprise/read-only-secrets-1.png)
+
+Secrets will now have a lock icon to show that they cannot be edited from Kestra, and the **Create New Secret** button in the top right corner that would otherwise be present is unavailable.
+
+![read-only-secrets-4](/docs/enterprise/read-only-secrets-4.png)
+
+To configure globally, add `readOnly: true` to the configuration of your external secret manager like in the examples below.
+
+### AWS Secret Manager
+ 
+The following example shows the configuration for AWS Secret Manager with read-only:
+
+```yaml
+kestra:
+  secret:
+    type: aws-secret-manager
+    readOnly: true
+    awsSecretManager:
+      accessKeyId: mysuperaccesskey
+      secretKeyId: mysupersecretkey
+      sessionToken: mysupersessiontoken
+      region: us-east-1
+```
+
+### Azure Key Vault
+
+The following example shows the configuration for Azure Key Vault with read-only:
+
+```yaml
+kestra:
+  secret:
+    type: azure-key-vault
+    readOnly: true
+    azureKeyVault:
+      clientSecret:
+        tenantId: "id"
+        clientId: "id"
+        clientSecret: "secret"
+```
+
+### Google Secret Manager
+
+The following example shows the configuration for Google Secret Manager with read-only:
+
+```yaml
+kestra:
+  secret:
+    type: google-secret-manager
+    readOnly: true
+    googleSecretManager:
+      project: gcp-project-id
+      serviceAccount: |
+        Paste here the contents of the service account JSON key file
+```
+
+### Vault
+
+With [Vault](./secrets-manager.md#vault-configuration), secrets are stored in a unique structure that can vary depending on the organization and version of Vault. Typically, there is a Secret Engine that hosts different Secrets with specific paths. Those Secrets are the paths to subkeys that are the actual key value pairs such as Username or Password to a service (e.g., `MY_SECRET = MY_SECRET_PASSWORD`). The following demonstrates a visual representation of this structure:
+
+```
+secret/
+  ├── app1/
+  │   ├── db/ <-- SECRET
+  │   │   ├── DATBASE_USERNAME    # Subkey
+  │   │   ├── DATABASE_PASSWORD   # Subkey
+  │   ├── api/ <-- SECRET
+  │       ├── keys                # Subkey
+  │       ├── API_TOKEN           # Subkey
+  ├── app2/
+      ├── config
+```
+
+- `secret`: This is the secret engine.
+- `app1` and `app2`: These are the path names to the secrets. This could be for example separate business units or applications.
+- `db`, `api`, and `config`: These are the secret names visible in the Kestra UI. `api` could be the Vault Secret that contains all API Keys for an application's external services.
+- `DATBASE_USERNAME`, `DATABASE_PASSWORD`, `keys`, `API_TOKEN`: These are the `subkey` key value pairs that can be used in a Kestra flow.
+
+With the above example structure, if we only need secrets for `app1`, our configuration in Kestra looks as follows with the added property `secretPathPrefix`:
```

Comment:
Maybe a note about how the prefix is not going to be used in the global config? probably not in the tenant one but check